atomic_reactor/plugins/pre_download_remote_source.py

```diff
@@ -13,8 +13,9 @@
 import base64
 import os
 import tarfile
+from six.moves import shlex_quote as quote
```

Comment:
https://github.com/containerbuildsystem/atomic-reactor/pull/1565  six was removed

Reply:
Taken from https://github.com/containerbuildsystem/dockerfile-parse/blob/master/dockerfile_parse/parser.py#L18
Otherwise unit tests for eentos7 will fail.

Reply:
What unittests on centos7? Rebase

Reply:
Ou that was it. Fixed